tests/vectordbs/test_qdrant.py

```diff
@@ -0,0 +1,107 @@
+"""Tests for verifying if the `qdrant` vector DB server is up and running."""
+
+import numpy as np
+import pytest
+from qdrant_client import QdrantClient
+from qdrant_client.models import Distance, PointStruct, VectorParams
+
+
+@pytest.fixture(name="client")
+def client_fixture():
+    """Fixture to get QdrantClient. Please run qdrant server with
+
+    ```
+    docker pull qdrant/qdrant
+    docker run -p 6333:6333 qdrant/qdrant
```

Comment:
Add `--rm` to the instructions so that the container is automatically deleted when it is stopped.